saas/{{cookiecutter.project_name}}/webapp/src/routes/+layout.ts

```diff
@@ -1,5 +1,7 @@
 import { features, loadFeatures } from '$lib/features';
 
+export const ssr = false;
```

Comment:
this should totally go under a feature flag... maybe SPA?

Reply:
I have a question about this.

Some of the checks we perform (such as checking for the keyring, but also feature flags) must be made in the browser.

But, because of SSR, a weird thing happens: the page is rendered, sent to the browser and displayed, and then if the browser check fails, the user is sent away. So the pages "flashes" and then goes away. imo a bit ugly (and also the user may see some content it's not allowed to).

So imo the app should always be full `ssr = false`